### DISCUSSION TITLE
Google Drive Movies

### DISCUSSION THREAD

**Discussion by connorm71909** [community/community#66086] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

For the website you made with the movies-list embedded in it with links to movies that take you too your google drive, do you know where someone would find more of those?

**Comment by Akash1134** [comment]
Hi @connorm71909 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
